### PR TITLE
[DOCS-6269] Search services performance recommendations 

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by oliveralfresco2018 — COMMENTED** [review]
Made changes as per review

**Review by eliaporciani — APPROVED** [review]

**Review by eliaporciani — APPROVED** [review]

File: _data/toc/insight-engine.yaml [hunk 1]
```diff
@@ -39,6 +39,8 @@
           path: '/insight-engine/latest/config/filtered/'
         - title: 'Transactional metadata query'
           path: '/insight-engine/latest/config/transactional/'
+        - title: 'Performance Recommendations'
+          path: '/search-services/latest/config/performance/'
```
Comment (gravitonian) [inline _data/toc/insight-engine.yaml:43]: Should this link go to search-services even though we are in insight-engine? The /insight-engine/latest/config/performance/ page does exist...

File: search-services/latest/config/performance.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+* [Disable FINGERPRINT](#disable-fingerprint) reduces storage requirements
+* [Disable SOLR Document Cache](#disable-solr-document-cache) reduces RAM requirements
+* [Optimize SOLR Index](#optimize-solr-index) improves search performance
+
+Additionally, from version 2.0.2, SOLR Merging parameters have been exposed that can be configured within the `solrcore.properties` file. The default values function adequately for many use cases, however some recommendations are given to increase performance in large deployments.
+
+## Disable FINGERPRINT
+
+The [Document Fingerprints]({% link search-services/latest/admin/index.md %}#disable-fingerprint) feature can be used to get similar documents from SOLR using the reserved word `FINGERPRINT` in `FTS` search syntax. In order to provide these results, each document in the SOLR Index includes a list of `MINHASH` fields that create larger Lucene Indexes.
```
Comment (gravitonian) [inline search-services/latest/config/performance.md:17]: Link (search-services/latest/admin/index.md %}#disable-fingerprint) seems to be incorrect?

File: insight-engine/latest/config/performance.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Additionally, from version 2.0.2, SOLR Merging parameters have been exposed that can be configured within the `solrcore.properties` file. The default values function adequately for many use cases, however some recommendations are given to increase performance in large deployments.
+
+## Disable FINGERPRINT
+
+The [Document Fingerprints]({% link insight-engine/latest/admin/index.md %}#document-fingerprints).feature can be used to get similar documents from SOLR using the reserved word `FINGERPRINT` in `FTS` search syntax. In order to provide these results, each document in the SOLR Index includes a list of `MINHASH` fields that create larger Lucene Indexes.
+
+From version 2.0.2 the Fingerprint feature is disabled by default and appears in the `solrcore.properties` file as `alfresco.fingerprint=false`.
+
+> ***Note:** This configuration will generate smaller Lucene indexes when indexing and may help to reduce storage and to increase performance.
```
Comment (gravitonian) [inline insight-engine/latest/config/performance.md:21]: Looks like one too many * (i.e. ***Note...)

File: search-services/latest/config/performance.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+From version 2.0.2 the Fingerprint feature is disabled by default and appears in the `solrcore.properties` file as `alfresco.fingerprint=false`.
+
+> ***Note:** This configuration will generate smaller Lucene indexes when indexing and may help to reduce storage and to increase performance.
```
Comment (gravitonian) [inline search-services/latest/config/performance.md:21]: Looks like one too many * (i.e. ***Note...)

File: search-services/latest/admin/index.md [hunk 5]
```diff
@@ -76,7 +76,9 @@ You can set different debug logging levels for Solr components using the Solr lo
 
 ### Document fingerprints
 
-Alfresco Content Services 6.2 provides support for Document Fingerprints to find related documents. Document Fingerprinting is performed by algorithms that map data, such as documents and files to shorter text strings, also known as fingerprints. This feature is exposed as a part of the Alfresco Full Text Search Query Language.
+Alfresco Content Services 6.2 and above provides support for Document Fingerprints to find related documents. Document Fingerprinting is performed by algorithms that map data, such as documents and files to shorter text strings, also known as fingerprints. This feature is exposed as a part of the Alfresco Full Text Search Query Language.
+
+> **NOTE:** Document fingersprints is disabled by default, for more see [Document Fingerprints]({% link search-services/latest/config/performance.md %}#document-fingerprints)
```
Comment (gravitonian) [inline search-services/latest/admin/index.md:81]: Looks like incorrect link, should it no go to disabled info...

**Review by gravitonian — CHANGES_REQUESTED** [review]
Some possible link problems

File: _data/toc/insight-engine.yaml (hunk 1 above)
Comment (oliveralfresco2018) [reply _data/toc/insight-engine.yaml:43]: no it shouldn't !

File: search-services/latest/config/performance.md (hunk 2 above)
Comment (oliveralfresco2018) [reply search-services/latest/config/performance.md:17]: @eliaporciani can you confirm if this link is correct please?
I presume I got it from you?
Cheers :-)

File: insight-engine/latest/config/performance.md (hunk 3 above)
Comment (oliveralfresco2018) [reply insight-engine/latest/config/performance.md:21]: fixed. thanks

File: search-services/latest/admin/index.md (hunk 5 above)
Comment (oliveralfresco2018) [reply search-services/latest/admin/index.md:81]: @eliaporciani can you confirm if this link is correct please?
I presume I got it from you?
Cheers :-)

File: search-services/latest/config/performance.md (hunk 4 above)
Comment (oliveralfresco2018) [reply search-services/latest/config/performance.md:21]: fixed.

File: search-services/latest/config/performance.md (hunk 2 above)
Comment (oliveralfresco2018) [reply search-services/latest/config/performance.md]: Done

File: search-services/latest/admin/index.md (hunk 5 above)
Comment (oliveralfresco2018) [reply search-services/latest/admin/index.md]: done

**Comment by oliveralfresco2018** [comment]
Corrected the links

File: search-services/latest/admin/index.md (hunk 5 above)
Comment (gravitonian) [reply search-services/latest/admin/index.md]: Link is broken: http://localhost:4000/search-services/latest/config/performance.md#disable-fingerprint

**Review by gravitonian — CHANGES_REQUESTED** [review]
Link still broken

File: search-services/latest/admin/index.md (hunk 5 above)
Comment (oliveralfresco2018) [reply search-services/latest/admin/index.md]: Cheers Martin, the link says this tho [Document Fingerprints]({% link search-services/latest/config/performance.md %}#document-fingerprints)

It should say **document-fingerprints** not disable?

It works on my local. The link you're quoting says **disable-fingerprint**   .

Comment (oliveralfresco2018) [reply search-services/latest/admin/index.md]: Yeah just checked the link on my local and all links work on that page, sorry can you check it again?
cheers.

Comment (anxumalo) [reply search-services/latest/admin/index.md]: See https://github.com/Alfresco/docs-alfresco/pull/317/files#diff-fd15bed585d8580ea08e53bd63f7d649b6ad120c8d09586e00d6fd84fb00ea56R101

The text says: 
> **NOTE:** Document fingersprints is disabled by default, for more see [Document Fingerprints]({% link insight-engine/latest/config/performance.md %}#disable-fingerprint)

1. `#disable-fingerprint` should be `#disable-document-fingerprint` based on the sections in `insight-engine/latest/config/performance.md`. The link works but the page anchor is ignored.
2. Typo: `fingersprints`.

Comment (oliveralfresco2018) [reply search-services/latest/admin/index.md]: All done cheers.

**Review by gravitonian — APPROVED** [review]
Looks ok now.

**Pull request merged**